force-app/main/selector/ContactSelector.cls

```diff
@@ -167,6 +168,18 @@ public inherited sharing class ContactSelector {
                 .build();
     }
 
+    @TestVisible
```

Comment:
Are the changes in this file still need it? If so can they be moved to the contact merge selector?